eliminates unused variable ROOT, which disturbs ld from Prefix. Bug #414335
--- Makefile.in	2001-11-27 07:38:13.000000000 +0900
+++ Makefile.in.new	2012-05-02 16:13:10.000000000 +0900
@@ -46,13 +46,12 @@ BINDIR=@sbindir@
 MANDIR=@mandir@
 RM=rm -f
 
-ROOT=./
-ANALYZE_SRCDIR=$(ROOT)base/
+ANALYZE_SRCDIR=base/
 COMMONDIR=common/
 COMMON_LIB_SRCDIR=$(COMMONDIR)lib/
-ANALYZE_NULL_SRCDIR=$(ROOT)null/
-ANALYZE_SSL_SRCDIR=$(ROOT)ssl/
-ANALYZE_RECORD_SRCDIR=$(ROOT)@RECORD_MOD@/
+ANALYZE_NULL_SRCDIR=null/
+ANALYZE_SSL_SRCDIR=ssl/
+ANALYZE_RECORD_SRCDIR=@RECORD_MOD@/
 include rules.mk
 
 include $(COMMON_LIB_SRCDIR)/targets.mk
